### DISCUSSION TITLE
k8s deploy

### DISCUSSION THREAD

**Discussion by echo-initial** [milvus-io/milvus#26546] [Q&A and General discussion]
etcd exception
![image](https://github.com/milvus-io/milvus/assets/127096463/1684ce2c-73a4-4528-9e9f-24a64900912d)

**Comment by LoveEachDay** [comment]
@echo-initial Could you provide the log for `etcd-0`?

**Reply by echo-initial** [reply]
![image](https://github.com/milvus-io/milvus/assets/127096463/6f62bf02-9d2d-45e9-a313-16579d5a1eb8)
